src/System.Net.Mail/tests/Functional/AttachmentTest.cs

```diff
@@ -32,6 +32,23 @@ public void ConstructorNullName()
             Assert.Equal(null, attach.Name);
         }
 
+        [Fact]
+        public void ConstructorPathName()
+        {
+            string attachFile = Path.GetTempFileName();
+            try
+            {
+                var fileStream = File.Create(attachFile);
```

Comment:
Change 'var' to the actual type.  It isn't obvious what that type should be.

Reply:
ok